File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/UploadChunkVO.kt
package com.sdkwork.app

data class UploadChunkVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val uploadId: String? = null,
    val chunkIndex: Int? = null,
    val chunkSize: Int? = null,
    val status: String? = null,
    val success: Boolean? = null,
    val errorMsg: String? = null,
    val chunkMd5: String? = null,
    val uploadedSize: Int? = null,
    val remainingChunks: Int? = null,
    val progress: Int? = null
)

File: sdkwork-app-sdk-typescript/src/types/upload-chunk-vo.ts
/** 上传分块响应 */
export interface UploadChunkVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 上传ID */
  uploadId?: string;
  /** 分块索引 */
  chunkIndex?: number;
  /** 分块大小 */
  chunkSize?: number;
  /** 上传状态 */
  status?: string;
  /** 上传结果 */
  success?: boolean;
  /** 错误信息 */
  errorMsg?: string;
  /** 分块MD5 */
  chunkMd5?: string;
  /** 已上传的大小 */
  uploadedSize?: number;
  /** 剩余块数 */
  remainingChunks?: number;
  /** 完成百分比 */
  progress?: number;
}
